app/styles/main.css

```diff
@@ -11,6 +11,14 @@ body {
   background: #ddd;
 }
 
+.tool-bar {
+  text-align: center;
+  font-size: 20px;
+  padding: 5px;
+  background-color: darkslategray;
+  color: white;
+}
+
```

Comment:
Lets move any styles meant for the header, in the header directive styles, Lets remove this here